container_workflow_tool/main.py

```diff
@@ -545,11 +556,14 @@ def dist_git_changes(self, rebase=False):
         self.distgit.dist_git_changes(images, rebase)
         self.logger.info("\nGit location: " + tmp)
         if self.args:
-            template = "./rebuild-helper {} git show"
+            tmp_str = ' --tmp ' + self.tmp_workdir if self.tmp_workdir else '"'
             self.logger.info("You can view changes made by running:")
-            self.logger.info(template.format('--base ' + self.base_image + (' --tmp ' + self.tmp_workdir if self.tmp_workdir else "")))
+            self.logger.info(f"cwt|rh_cwt --base {self.base_image} {tmp_str} git show")
```

Comment:
I would say using rhcwt here would just confuse anyone not knowing of its existence :)